### PR TITLE
Strong modeify the examples

### PR DESCRIPTION
**Pull request by Hixie**
This makes skyanalyzer also check the examples, and fixes everything it
found there.

### CODE REVIEW

**Comment by Hixie** [comment]
cc @abarth 

cc @vlidholt this touches game/ code

File: examples/fitness/lib/meal.dart [hunk 1]
```diff
@@ -85,10 +85,11 @@ class MealFragmentState extends State<MealFragment> {
 
   Widget buildBody() {
     Meal meal = new Meal(when: new DateTime.now());
+    // TODO(ianh): Fix Block such that we could use that here instead of rolling our own
```
Comment (abarth) [inline examples/fitness/lib/meal.dart]: I'm pretty sure we can.  `Block` didn't used to have padding, but now it does.

**Comment by abarth** [comment]
LGTM

**Comment by Hixie** [comment]
Github says Travis is unhappy, but the logs say otherwise, so I'm landing this...

**Pull request merged**